src/Application.php

```diff
@@ -215,7 +218,7 @@ private function getBaseCacheDir()
     {
         $baseCacheDir = Path::create(
             [
-                \sys_get_temp_dir(),
+                $this->cacheDirectoryFactory->generate()->toString(),
```

Comment:
Let's utilize `CacheDirectoryFactory` fully and move adding `.crunz` to `generate` method

Reply:
I think, try with always added `.crunz`: `CRUNZ_BASE_CACHE_PATH=/tmp/crunz` should produce `/tmp/crunz/.crunz`

Reply:
Okay, I also added this information to the README fie